metadata-models/src/main/pegasus/com/linkedin/metadata/relationship/EvaluatedOn.pdl

```diff
@@ -0,0 +1,16 @@
+namespace com.linkedin.metadata.relationship
+
+/**
+ * A generic model for the Evaluated-On relationship
+ */
+@pairings = [ {
+  "destination" : "com.linkedin.common.urn.DatasetUrn",
+  "source" : "com.linkedin.common.urn.MLModelUrn"
+}]
+record EvaluatedOn includes BaseRelationship {
+
+  /**
+   * How was the data preprocessed for evaluation (e.g., tokenization of sentences, cropping of images, any filtering such as dropping images without faces)?
+   */
+  preProcessing: optional string
```

Comment:
Could you share a usecase where storing this property in the edge will be useful? I wouldn't recommend adding this field as an edge property.
cc @keremsahin1 @camelliazhang

Reply:
I've removed it. My thought was essentially that it would be useful information to return as a result of searching for the relationship or perhaps filtering on certain types of preprocessing that had occurred similar to how OwnedBy has OwnershipType. If the issue is that it's a arbitrary string I don't currently have a list of reasonable preprocessing examples for an enum so we can leave it off and add it later if it's worthwhile.